mevislab.github.io/content/tutorials/testing/testingexample3.md

```diff
@@ -18,26 +18,28 @@ In this example, you are writing an iterative test. Iterative test functions run
 
 ## Steps to do
 ### Creating the network to be used for testing
-Add a `LocalImage`and a `DicomTagViewer` module to your workspace and connect them. Save the network.
+Add a `LocalImage` and a `DicomTagViewer` module to your workspace and connect them.
 
 ![Example Network](/images/tutorials/testing/network_test3.png "Example Network")
 
 ### TestCase creation
-Start MeVisLab TestCaseManager and create a new TestCase called *IterativeTestCase* as seen in [Example 1: Writing a simple testcase in MeVisLab](/tutorials/testing/testingexample1).
-
 Open the panel of the `DicomTagViewer` and set *Tag Name* to *WindowCenter*. The value of the DICOM tag from the current input image is automatically set as value.
 
+Save the network.
+
+Start MeVisLab TestCaseManager and create a new TestCase called *IterativeTestCase* as seen in [Example 1: Writing a simple testcase in MeVisLab](/tutorials/testing/testingexample1).
```

Comment:
Actually "TestCase" is not a word, just use "test case" (some above: "Test case creation")

Reply:
Still "TestCase creation"